clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBIngestThrottler.java

```diff
@@ -0,0 +1,64 @@
+package com.linkedin.davinci.store.rocksdb;
+
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.Lock;
+import java.util.concurrent.locks.ReentrantLock;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.rocksdb.RocksDBException;
+
+
+/**
+ * This class is a copy of {@link RocksDBOpenThrottler}
+ * check {@link RocksDBServerConfig#ROCKSDB_DB_INGEST_OPERATION_THROTTLE} to find more details.
+ */
+public class RocksDBIngestThrottler {
```

Comment:
There are a lot of duplicate code between this class and `RocksDBOpenThrottler`, so can we extract a common class?

Reply:
Cleaned this up.